<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmProgressBarRound</class>
 <widget class="QWidget" name="frmProgressBarRound">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>圆环风格</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>饼状风格</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>线条风格</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="ProgressBarRound" name="roundBar1" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="ProgressBarRound" name="roundBar2" native="true"/>
   </item>
   <item row="1" column="2">
    <widget class="ProgressBarRound" name="roundBar3" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="ProgressBarRound" name="roundBar4" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="ProgressBarRound" name="roundBar5" native="true"/>
   </item>
   <item row="2" column="2">
    <widget class="ProgressBarRound" name="roundBar6" native="true"/>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QSlider" name="slider">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>300</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>25</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProgressBarRound</class>
   <extends>QWidget</extends>
   <header>progressbarround.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
